matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -19,6 +19,13 @@ var PhenoTips = (function (PhenoTips) {
     {
         this._tableElement = $('matchesTable');
         this._ajaxURL = XWiki.contextPath + "/rest/matches";
+
+        this._loadMatchesURL = this._ajaxURL;
+        this._onSimilarCasesPage = $('similar-cases-container');
```

Comment:
I would rename all reference to "similarCasesPage" to simply "patientPage". This is the first time we introduce this name\word in this code, lets not use legacy names which will be completely unclear in the future, since `SimilarCases.xml` is removed in the same PR (so in the future there will be a question "what the hell is 'similar cases'"?

Reply:
Also IMHO the check for that page can be more robust than checking `$('similar-cases-container')` - e.g. check that space is data, which is less likely to change

Reply:
SimilarCases.xml is not removed